test/integrations/validation/Validator.spec.ts

```diff
@@ -226,6 +226,62 @@ describe('Validator', () =>
         });
     });
 
+    describe('Url values', () =>
+    {
+        it('should reject an invalid url with protocol', () =>
+        {
+            const input = { url: 'http://this is an invalid url' };
+            const test = () => validator.validate(input, SCHEMAS.URL_HTTP_HTTPS);
+
+            expect(test).toThrowError(ERRORS.URL);
+        });
+
+        it('should reject a url without protocol', () =>
+        {
+            const input = { url: 'masking.tech/images/peter.jpg' };
+            const test = () => validator.validate(input, SCHEMAS.URL_HTTP_HTTPS);
+
+            expect(test).toThrowError(ERRORS.URL);
+        });
+
+        it('should accept a valid url when no protocols are configured', () =>
+        {
+            const input = { url: 'someprotocol://masking.tech/images/peter.jpg' };
+            const result = validator.validate(input, SCHEMAS.URL_NO_PROTOCOL);
+
+            expect(result).toBeUndefined();
+        });
+
+        it('should accept a valid https url', () =>
+        {
+            const httpInput = { url: 'https://masking.tech/images/peter.jpg' };
+            const httpsResult = validator.validate(httpInput, SCHEMAS.URL_HTTP_HTTPS);
+
+            expect(httpsResult).toBeUndefined();
+        });
+
+        it('should reject an url with protocol not configured', () =>
+        {
+            const FtpInput = { url: 'ftp://masking.tech/images/peter.jpg' };
+            const FtpTest = () => validator.validate(FtpInput, SCHEMAS.URL_HTTP_HTTPS);
+
+            expect(FtpTest).toThrowError(ERRORS.URL);
+
+            const HttpInput = { url: 'http://masking.tech/images/peter.jpg' };
+            const HttpTest = () => validator.validate(HttpInput, SCHEMAS.URL_HTTPS_FTP);
+
+            expect(HttpTest).toThrowError(ERRORS.URL);
+        });
+
+        it('should reject a too long url', () =>
+        {
+            const input = { url: 'https://' + 'masking.tech/images'.repeat(110) + '/peter.jpg' };
```

Comment:
```suggestion
            const folder = 'folder_with_a_long_name/';
            const input = { url: `https://example.com/${folder.repeat(110)}` };
```
I think this version makes it more readable.